src/categorizer.py

```
@@ -26,146 +28,668 @@ def __init__(
+            },
+            "performance_metrics": {
+                "tokens_per_second": 0.0,
+                "cost_per_ticket": 0.0,
+                "avg_tokens_per_chunk": 0.0,
+                "avg_tokens_per_batch": 0.0
+            },
+            "session_start_time": time.time(),
+            "budget_alerts": []
+        }
+        
+        print(f"💰 Sistema de Token Tracking configurado:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)